src/main/java/com/work/management/web/rest/controller/EmployeeController.java

```diff
@@ -0,0 +1,35 @@
+package com.work.management.web.rest.controller;
+
+import com.work.management.entity.Employee;
+import com.work.management.service.DataPersistingService;
+import com.work.management.web.rest.resource.EmployeeResource;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "workmanagement/v1/employees")
+public final class EmployeeController {
```

Comment:
Try removing public from all controllers class and their constructors also... make sure you limit class visibility eveywhere possible....